dipy/segment/tests/test_refine_rb.py

```diff
@@ -0,0 +1,172 @@
+import numpy as np
```

Comment:
@BramshQamar if this test file is exactly the same as the RB test file + calling refine then let's just keep one of the two. Because otherwise we are just replicating code.

Reply:
Should this be done before merging?

Reply:
It's already done.